### PR TITLE
CA-191570: Make use of meaningful tokens during xenstore watch

### PR DESCRIPTION
**Pull request by koushikcgit**
Signed-off-by: Koushik Chakravarty <koushik.chakravarty@citrix.com>

### CODE REVIEW

File: xc/xenops_server_xen.ml [hunk 1]
```diff
@@ -2440,7 +2442,7 @@ module Actions = struct
 		device_watches := IntMap.add domid [] !device_watches
 
 	let domain_disappeared xc xs domid =
-		let token = Xenstore_watch.token_of_domain domid in
+		let token = Printf.sprintf "xenopsd-xc:domain-%d" domid in
```
Comment (robhoes) [inline xc/xenops_server_xen.ml]: Why not update `Xenstore_watch.token_of_domain` to return the new string? Now we have the same hardcoded string in many places.

**Comment by robhoes** [comment]
I think this looks good after the hardcoded strings have been factored out.

**Comment by koushikcgit** [comment]
@robhoes Updated

**Comment by robhoes** [comment]
Thanks. :shipit:

**Pull request merged**